app/mailers/comment_notification_mailer.rb

```diff
@@ -1,22 +1,36 @@
 class CommentNotificationMailer < ApplicationMailer
-  def email_notification(comment)
+
+  def reviewer_notification(comment, reviewers)
```

Comment:
This is definitely on the nitpick level and it's not a big deal but it jumped out at me.  The reviewer mailer method takes comment and reviewers while the speak mailer method takes proposal, comment and speakers.  I'd make both signatures the same.  Since you have access to the proposal already in verify, you may as well pass it in.  It then makes both reviewer and speaker mailer method signatures the same.